docs/runbooks/cloud9/cloud9_init.sh

```diff
@@ -66,16 +67,18 @@ function postgresql() {
   local environment_name=$(echo ${1:?} | awk '{print tolower($0)}')
   export AWS_DEFAULT_REGION=eu-west-1
   export PGHOST=$(aws rds describe-db-instances --db-instance-identifier api-${environment_name} | jq -r .'DBInstances'[0].'Endpoint'.'Address')
```

Comment:
An interesting optimisation possible here. The script is now making the exact same api call and using jq to grab the right part to set with export.

If you make the request once and assign it to a variable like db_instance and then use jq against that variable to export the bits we need.

```
db_instance=$(aws rds describe-db-instances --db-instance-identifier api-${environment_name}
  export PGHOST= $(cat $db_instance | jq -r .'DBInstances'[0].'Endpoint'.'Address')
️```
or something like that

Reply:
good point - will take a look